### PR TITLE
Fix Ntop NG hang on stop/restart. Issue #11185

### PR DESCRIPTION
**Pull request by dragoangel**

### CODE REVIEW

File: net/pfSense-pkg-ntopng/files/usr/local/pkg/ntopng.inc [hunk 1]
```diff
@@ -155,7 +155,7 @@ function ntopng_sync_package() {
 	/* Create rc script */
 	$stop = <<<EOD
 # Kill ntopng and redis
-	/usr/bin/killall ntopng redis-cli redis-server
+	while [ "$(pgrep ntopng)" != "" ]; do /usr/bin/killall ntopng && sleep 3; done; /usr/bin/killall redis-cli redis-server
```
Comment (rbgarga) [inline net/pfSense-pkg-ntopng/files/usr/local/pkg/ntopng.inc:158]: Please use pgrep's -q parameter and also split it in multiple lines to improve readability:
```
while pgrep -q ntopng; do
    /usr/bin/killall ntopng
    sleep 3
done
/usr/bin/killall redis-cli redis-server
```

**Review by rbgarga — CHANGES_REQUESTED** [review]

File: net/pfSense-pkg-ntopng/files/usr/local/pkg/ntopng.inc (hunk 1 above)
Comment (dragoangel) [reply net/pfSense-pkg-ntopng/files/usr/local/pkg/ntopng.inc:158]: Ok, np 😄

Comment (dragoangel) [reply net/pfSense-pkg-ntopng/files/usr/local/pkg/ntopng.inc]: Checked on my own instance before commit*

**Review by rbgarga — CHANGES_REQUESTED** [review]
In order to get a new package built PORTVERSION or PORTREVISION must be bumped at port Makefile

**Comment by dragoangel** [comment]
I not done this to provide you ability to manage this in terms of do release with other commits or not.

**Comment by dragoangel** [comment]
Can I increase version in makefile?

**Comment by rbgarga** [comment]
> Can I increase version in makefile?

yes, please

**Comment by dragoangel** [comment]
recreated  in #1030

**Pull request closed without merging**